internal/pkg/services/argus/utils/utils_test.go

```diff
@@ -220,3 +349,87 @@ func TestValidatePlanId(t *testing.T) {
 		})
 	}
 }
+
+func TestMapToUpdateScrapeConfigPayload(t *testing.T) {
```

Comment:
The other smaller functions that are called from this one should also be tested (apart maybe from the ones that are just a simple assignment to another data structure)